modules/gapi/perf/common/gapi_video_perf_tests_inl.hpp

```diff
@@ -241,6 +241,150 @@ PERF_TEST_P_(BackgroundSubtractorPerfTest, TestPerformance)
 
 //------------------------------------------------------------------------------
 
+PERF_TEST_P_(KalmanFilterControlPerfTest, TestPerformance)
+{
+    MatType2 type = -1;
+    int  dDim = -1, mDim = -1;
+    std::size_t testNumMeasurements = 0;
+    cv::GCompileArgs compileArgs;
+    std::tie(type, dDim, mDim, testNumMeasurements, compileArgs) = GetParam();
+
+    const int cDim = 2;
+    cv::gapi::KalmanParams kp;
+    initKalmanParams(kp, type, dDim, mDim, cDim);
+
+    // Generating input
+    std::vector<bool> haveMeasurements;
+    std::vector<cv::Mat> measurements, ctrls;
+    measurements.reserve(testNumMeasurements);
+    ctrls.reserve(testNumMeasurements);
+    haveMeasurements.reserve(testNumMeasurements);
+    cv::RNG& rng = cv::theRNG();
+    {
+        cv::Mat measurement(mDim, 1, type);
+        cv::Mat ctrl(cDim, 1, type);
+        for (std::size_t i = 0; i < testNumMeasurements; i++)
+        {
+            haveMeasurements[i] = rng(2u); // returns 0 or 1 - whether we have measurement
+                                           // at this iteration or not
+            if (haveMeasurements[i] == true)
```

Comment:
Without `== true`

Reply:
It didn't work for some reason, so I reinsured and did it everywhere.. But actually only our `cv::gin()` requires not to put `haveMeasurements[i]` into it. Idk why, maybe it is an issue with the `GOpaque`, I will ask the team
Will change where changeable, thanks

Reply:
Fixed